web/Web/web/static/js/document-viewer.js

```diff
@@ -1064,6 +1067,16 @@ docView.prototype = {
       return "";
     }
 
+    function checkURL(str) {
```

Comment:
Minor edit. Change the name of this function to `isURL`

Reply:
I just made the changes.